### DISCUSSION TITLE
Stopping a NanoLimbo server doesn't seem to work

### DISCUSSION THREAD

**Discussion by BBaoVanC** [itzg/docker-minecraft-server#1294] [Q&A]
I'm not sure whose end this is an issue on, but I have a [NanoLimbo](https://github.com/Nan1t/NanoLimbo) server running which doesn't stop with `docker compose stop` until it gets SIGKILLed. NanoLimbo doesn't seem to support rcon, but it does stop when sending ctrl-c while running. How can I get it to stop on demand?

**Comment by itzg** [comment]
Can you provide the container logs during shutdown? The server runner is supposed to fallback to sending a `stop` command to the server's stdin if rcon didn't work. If nano limbo doesn't support either graceful shutdown mechanism the I need details for an enhancement.

**Reply by BBaoVanC** [reply]
Log looks like this:

```
mc-limbo-1  | 2022/01/24 01:03:28 Stopping with rcon-cli
mc-limbo-1  | 2022-01-24T01:03:28.268Z	ERROR	mc-server-runner	Failed to stop using rcon-cli	{"error": "exit status 1"}
mc-limbo-1  | 2022-01-24T01:03:28.269Z	INFO	mc-server-runner	Sending 'stop' to Minecraft server...
mc-limbo-1  | 2022-01-24T01:03:28.269Z	INFO	mc-server-runner	Waiting for completion...
```

I tested it outside of Docker and it doesn't respond to `stop` or `end` or any command at all. Only Ctrl-C or a SIGTERM works.

**Reply by BBaoVanC** [reply]
Update: there's now a pull request to NanoLimbo that I have compiled which adds support for a `stop` command. It would still nice to have this Docker image support sending a SIGTERM in case the stop command isn't supported, maybe this could be configured through an environment variable?
